.conform.yaml

```diff
@@ -0,0 +1,14 @@
+policies:
+  - type: commit
+    spec:
+      conventional:
+        types:
+          - chore
```

Comment:
```suggestion
          - upkeep
```
Stemmer for `upkeep`, da jeg ikke liker å tenke at arbeidet jeg holder på med er et `chore`.

Reply:
Stemmer for `chore` da det fremstår for meg som mer etablert, og trekkes frem som eksempel i docsa:

Nr 4 her: https://www.conventionalcommits.org/en/v1.0.0/#summary

Reply:
Enig med @sechmann her (og i den andre kommentaren over).

Reply:
Mente Angular hadde fjernet `chore` med noen lignende tanker som meg, men var på grunn av overforbruk 😅 

> Note that this PR removes support for "chore" which was previously overused by everyone on the team.
https://github.com/angular/angular/commit/dff6ee32725197bdb81f3f63c5bd9805f2ed22bb

Reply:
Det er kanskje er større bekymring ja :sweat_smile: